SMLHelper/modBZ.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
`modSN.json` and `modBZ.json`? Really?
Group these in a single folder like `QModManager/Subnautica.json` and `QModManager/BelowZero.json` or something